shell/pages/auth/verify.vue

```diff
@@ -25,21 +25,21 @@ function isSaml($route) {
 }
 
 export default {
-  async fetch({ store, route, redirect }) {
-    const code = route.query[GITHUB_CODE];
-    const stateStr = route.query[GITHUB_NONCE];
+  async fetch() {
```

Comment:
While working on this the verify step of the auth provider was also failing.

It became apparent that the [legacy fetch](https://v2.nuxt.com/docs/features/data-fetching/#the-fetch-hook) was no longer being invoked. I converted this to the new version of fetch to resolve this.

I found one other usage of the legacy fetch https://github.com/rancher/dashboard/blob/master/shell/pages/clusters/index.vue. That page does not appear to be getting used or linked to anywhere and I'd rather address it in a separate PR.

Reply:
Sounds good. It's just a helper to redirect `/c` to a cluster. I don't think it's used anywhere other than when users nav directly to that url